### DISCUSSION TITLE
Loading LLAMA 13B pytorch model is not working in windows, I want to fix it.

### DISCUSSION THREAD

**Discussion by akkinenirajesh** [tinygrad/tinygrad#994] [Q&A]
I am getting 
AttributeError: 'tuple' object has no attribute 'fxn'
at line #132 in utils.py 
`t.lazydata.op.arg.fxn = _mmap`

I have a (32000, 2560) tuple in arg. 

What is this fxn and how does it work?

**Comment by boringdanv** [comment]
yep, the same problem

**Reply by KjellRChristensen** [reply]
Have the same for Mac Book Pro M1 Max 64 Gb. Looking for the merge where George got this to load?

**Reply by KjellRChristensen** [reply]
I had a run at 7B on the master and it runs ok! For the 13B, you have to back track 4 months on the master to get the commit just after George managed to load the weights.  I have made som adjustments to llama.py and utils.py to get the 13B up and running. I have a Mac Book Pro M1 Max with 64GB. You need more than 32 to avoid swapping. Stacy is much more "fun" in 13B modus.

**Comment by wozeparrot** [comment]
This should have been fixed with #1275
